packages/autorest.go/test/synapse/azspark/zz_models.go

```diff
@@ -207,7 +206,9 @@ type Session struct {
 	// Dictionary of
 	AppInfo    map[string]*string
 	ArtifactID *string
-	Errors     []*ServiceError
+
+	// The error information.
+	Errors []*ServiceError
```

Comment:
I don't know why we didn't have this comment before, but I get the same diff on main.

Reply:
Hmm could this come from the version of autorest being used in CI?  We don't pin to a specific version.  If you update your autorest installation to latest and regenerate `azspark` does the comment show up?

Reply:
Hey, that did the trick 🏆